backend/src/test/resources/application-default.yaml

```diff
@@ -1,5 +1,5 @@
 spring:
-  profiles.include: no-security,no-okta-mgmt, no-okta-auth, no-experian
+  profiles.include: no-security,no-okta-mgmt, no-okta-auth, no-experian, disable-graphql-multipart
```

Comment:
Graphql (hacky) multipart support doesnt work during testing, this profile is here to disable it

Reply:
Can you add a comment here with a reminder to remove this profile once the REST endpoint for uploads is in place? I know you already have it on the `@Disabled` test but I think it'd be easy to forget to remove the profile when that work is happening.

Reply:
added a comment about it here: https://app.zenhub.com/workspaces/simplereport-better-data-team-605b43d51ea9f700119a48ee/issues/cdcgov/prime-simplereport/4266 feel free to remove comment if unnecessary 😸